python/dgl/typing.py

```diff
@@ -0,0 +1,25 @@
+"""DGL type hints"""
+
+__all__ = ["NData", "EData", "NType", "EType"]
+
+from typing import Tuple, Union
+
+try:
+    import torch
+
+    NData = torch.Tensor
+    EData = torch.Tensor
+except ImportError:
+    # When PyTorch is not available, create two class stubs to represent
```

Comment:
nit: there are still quite a lot of space in this line before hit 80, let's make use of them.